eox_hooks/admin.py

```diff
@@ -0,0 +1,21 @@
+"""
+Admin class.
+"""
+from django.contrib import admin
+
+from eox_hooks.models import HookExecutionAudit
+
+
+class HookExecutionAuditAdmin(admin.ModelAdmin):
+    """Audit admin."""
```

Comment:
Is there a way to make this admin class so that not even a super user can tamper with this DATA?

Reply:
I think we can override `has_add_permission` and `has_delete_permission` so they always return False